extensions/reviewed/BoidsMovement.json

```diff
@@ -387,6 +389,9 @@
                 "      sumY += otherObject.getCenterYInScene();",
                 "      count++;",
                 "    }",
+                "",
+                "    this.nearby = count;",
```

Comment:
I don't see anything using this value.  What is it used for?

Reply:
Oops I used it for debugging